examples/howtos/howto_ensemble_calculation.ipynb

```diff
@@ -0,0 +1,537 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# **SchnetPack Ensemble Calculator**"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Preparing and loading your data"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
```

Comment:
In the first section you should describe what to expect in this tutorial. you will show how to use the ensemble calculator its uncertainty measures and show examples for struc. relax. and MD, ...

Reply:
Describing the imported packages is not required